src/Reducers/rootReducer.js

```diff
@@ -28,8 +28,37 @@ const countyData = (state = "", action) => {
     }
 }
 
+const cycleFlag = (state = "", action) => {
+    switch(action.type) {
+      case 'TOGGLE_CYCLE_FLAG':
+        return !state
+      default:
+        return state
+    }
+}
+
+const cropImageName = (state = "", action) => {
+    switch(action.type) {
+      case 'CHANGE_CROP_IMAGE':
+      if(action.payload === 'hazelnut'){
+        console.log('changing to leaf')
```

Comment:
Generally it's bad to allow console.log's to get to production. They will slow down execution (which in this case it doesn't really matter). Furthermore, I assume this is for debugging purposes. If you're not using it already the Redux Dev Toolkit makes it really easy to see what state changed when an action is dispatched (and whether or not an action was dispatched at all). Making life much easier.